packages/loader/container-loader/src/container.ts

```diff
@@ -1619,11 +1619,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             if (errorMsg !== undefined) {
                 const error = new GenericError(
                     errorMsg,
+                    undefined,
                     {
                         clientId: this._clientId,
                         messageClientId: message.clientId,
                         sequenceNumber: message.sequenceNumber,
                         clientSequenceNumber: message.clientSequenceNumber,
+                        messageTimestamp: message.timestamp,
```

Comment:
@anthony-murphy , @agarwal-navin  - How helpful would it be if I converted this to a human readable datetime format rather than just the Unix epoch format?

Reply:
I am not sure that will help. We have a lot of telemetry where message timestamp is in Unix epoch format. Also, server logs, console logs, op stream, etc. would have it in that format. I would rather keep the same format so that its consistent.

Reply:
I'm fine with the epoch